app/Repositories/ReportbackRepository.php

```diff
@@ -134,4 +135,20 @@ public function getReportbackByUser($campaignId, $campaignRunId, $userId, $type
 
         return Reportback::where($parameters)->first();
     }
+
+    /**
+     * Updates an individual reportbackitem or many reportback items.
+     *
+     * @param array $data
+     *
+     * @return
+     */
+    public function updateReportbackItems($data)
+    {
+        foreach ($data as $reportbackItem) {
+            $rbItem = ReportbackItem::where(['id' => $reportbackItem['rogue_reportback_item_id']])->first();
+            $rbItem->status = $reportbackItem['status'];
+            $rbItem->save();
+        }
+    }
```

Comment:
This function should return something, either the updated item or null if something failed (i.e those properties above don't exist in the request)